### DISCUSSION TITLE
Setting navigation group as closed by default

### DISCUSSION THREAD

**Discussion by IllesAprod** [filamentphp/filament#2071] [Ideas]
Hi! We have an admin panel with more than ten models, which are fixtures like Country, Vat, Currency, etc... These models could be updated sometime but not too frequently. We have made a group for them called "Settings," but it still takes a lot of space in the sidebar. It would be nice if that group could be closed initially.

I haven't seen a method to do this right now, but I would be happy to PR this; if you can give me an idea, how would you do that?

**Comment by danharrin** [comment]
Hey! If I were to PR it, I'd add it as an option to the Advanced Navigation API - https://filamentphp.com/docs/2.x/admin/navigation#advanced-navigation-customization


Maybe a third parameter to `group()` - `bool isOpenByDefault = true`. Wdyt @ryangjchandler ?

**Comment by dev3k** [comment]
was looking for solution for this, so for now i'm using below temp solution

register custom js file with below content 

`
if (Alpine.store('sidebar').collapsedGroups.length === 0) {
    Alpine.store('sidebar').toggleCollapsedGroup('GROUP_NAME');
}
`

**Reply by danharrin** [reply]
Coincidence! Literally worked on this today - #3216

**Reply by danharrin** [reply]
You can pass `collapsed()` to a group when you register it in a service provider

**Reply by TheBlackJJoker** [reply]
Not work  for me. How i can to fix it?

**Reply by dev3k** [reply]
@TheBlackJJoker this was for v2, if you want it for V3 check [here](https://filamentphp.com/docs/3.x/panels/navigation#customizing-navigation-groups)

**Reply by TheBlackJJoker** [reply]
Yes, i know, but collapsed() not working. 

Menu is open when i login to the admin panel.

**Reply by danharrin** [reply]
It's for individual navigation groups, not the entire sidebar.

**Reply by TheBlackJJoker** [reply]
Yes, i know. I mean navigation goup collapsed do not work. 

Not sidebar

**Reply by danharrin** [reply]
Clear your browser local storage
